src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlyOtherAlone.java

```diff
@@ -145,12 +145,12 @@ class WithArraysAlone {
 }
 
 class Interface {
-    public @interface TestAnnotation {} //warn
+    public @interface TestAnnotation {}
 
-    public @interface TestAnnotation1 { String someValue(); } //warn
+    public @interface TestAnnotation1 { String someValue(); }
 
     public @interface TestAnnotation2 {
-        String someValue();} //warn
+        String someValue();}
```

Comment:
This stood out to me as weird that the warning goes away.

Reply:
We can double check by CLI, if it is not a violation - create issue

Reply:
```
$ cat Test.java 
class Interface {
    public @interface TestAnnotation {}

    public @interface TestAnnotation1 { String someValue(); }

    public @interface TestAnnotation2 {
        String someValue();}  // line 7, it should be warn

    public @interface TestAnnotation3 {
        String someValue();
    } //ok

    public @interface TestAnnotation4 { String someValue();
    } //ok
}

$ java -classpath checkstyle-10.2-all.jar com.puppycrawl.tools.checkstyle.Main -c /google_checks.xml Test.java 
Starting audit...
[WARN] /var/tmp/Test.java:1:1: The name of the outer type and the file do not match. [OuterTypeFilename]
[WARN] /var/tmp/Test.java:2:5: 'annotation def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[WARN] /var/tmp/Test.java:4:5: 'annotation def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[WARN] /var/tmp/Test.java:6:5: 'annotation def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[WARN] /var/tmp/Test.java:7:9: 'annotation field def modifier' has incorrect indentation level 8, expected level should be 4. [Indentation]
[WARN] /var/tmp/Test.java:7:27: ';' is not followed by whitespace. [WhitespaceAfter]
[WARN] /var/tmp/Test.java:7:28: WhitespaceAround: '}' is not preceded with whitespace. [WhitespaceAround]
[WARN] /var/tmp/Test.java:9:5: 'annotation def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[WARN] /var/tmp/Test.java:10:9: 'annotation field def modifier' has incorrect indentation level 8, expected level should be 4. [Indentation]
[WARN] /var/tmp/Test.java:11:5: 'annotation def rcurly' has incorrect indentation level 4, expected level should be 2. [Indentation]
[WARN] /var/tmp/Test.java:13:5: 'annotation def modifier' has incorrect indentation level 4, expected level should be 2. [Indentation]
[WARN] /var/tmp/Test.java:14:5: 'annotation def rcurly' has incorrect indentation level 4, expected level should be 2. [Indentation]
Audit done.
```

it is warn, so our test implementation matching real execution.

Reply:
we need to create issue on this.